server/src/main/java/org/apache/iotdb/db/qp/sql/IoTDBSqlVisitor.java

```diff
@@ -194,6 +194,11 @@ public class IoTDBSqlVisitor extends IoTDBSqlParserBaseVisitor<Operator> {
   // for create-cq clause and select-into clause.
   private static final Pattern leveledPathNodePattern = Pattern.compile("\\$\\{\\w+}");
 
+  /** used to match node name without special characters */
+  private static final String NODE_NAME_MATCHER = "([a-zA-Z0-9_:@#${}\\u2E80-\\u9FFF]+)";
```

Comment:
This is an important constant but duplicated in `ASTVisitor.java`. Could it be put into a common place?

Reply:
fixed. Removed to TsFileConstant.